### ISSUE TITLE
error when passing properties to createClass()

### ISSUE THREAD

**Issue by l2silver** [gaearon/babel-plugin-react-transform#88]
const properties = {...}

When I try to create a React.classCreate(properties), where properties is an object with the react class properties I want to include, I get the following error...

but, when I simply put the properties directly into React.classCreate({...}), there is no error

Module build failed: TypeError: /..../src/....js: Cannot read property 'find' of undefined
    at find (/..../node_modules/array-find/find.js:5:17)
    at getDisplayName (/..../node_modules/babel-plugin-react-transform/lib/index.js:33:44)
    at Object.CallExpression (/..../node_modules/babel-plugin-react-transform/lib/index.js:110:27)
    at NodePath._call (/..../node_modules/babel-traverse/lib/path/context.js:74:18)
    at NodePath.call (/..../node_modules/babel-traverse/lib/path/context.js:46:17)
    at NodePath.visit (/..../node_modules/babel-traverse/lib/path/context.js:104:12)
    at TraversalContext.visitQueue (/..../node_modules/babel-traverse/lib/context.js:156:16)
    at TraversalContext.visitSingle (/..../node_modules/babel-traverse/lib/context.js:113:19)
    at TraversalContext.visit (/..../node_modules/babel-traverse/lib/context.js:200:19)
    at Function.traverse.node (/..../node_modules/babel-traverse/lib/index.js:139:17)

**Comment by gaearon** [comment]
Does https://github.com/gaearon/babel-plugin-react-transform/pull/87 fix this?

**Comment by gaearon** [comment]
I think we just fixed this in `2.0.1`.
Please reopen if you still have an issue!

**Comment by nfcampos** [comment]
```js
const spec = {
  render: function () {}
};
React.createClass(spec);
```
a component like this won't be wrapped — but it also won't throw an error
(I don't see any good way to wrap components like this without generating a lot of false positives)

**Comment by l2silver** [comment]
Yeah, sorry, fixed > 2.0.0

Not sure if this matters, but this error was being generated with the babel-preset-react-hmre, which still needs to be updated for 2.02 dependency.
